<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinomialDistForm</class>
 <widget class="QWidget" name="BinomialDistForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="parametersGBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>321</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>321</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Binomial Distribution</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="nf1Label">
        <property name="text">
         <string>n</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="nSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="4">
       <widget class="QSlider" name="nHSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="pLabel">
        <property name="text">
         <string>p</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="pDSpinBox">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="4">
       <spacer name="pHSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="POLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>P(</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="kminSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="xLabel">
        <property name="font">
         <font>
          <family>Symbol [monotype]</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>£ C £</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="kmaxSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="PCLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QCheckBox" name="joinCheckBox">
        <property name="text">
         <string>Join</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="5">
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>31</height>
         </size>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="eqLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="BinomialDistPlotter" name="binPlotter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>281</width>
       <height>311</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="miscGBox">
     <property name="title">
      <string>Misc</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="exportPushButton">
        <property name="text">
         <string>&amp;Export to...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="csGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Coordinate Window</string>
     </property>
     <widget class="QLabel" name="xmaxLabel">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>26</y>
        <width>36</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>x-max</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="xmaxSpinBox">
      <property name="geometry">
       <rect>
        <x>53</x>
        <y>26</y>
        <width>59</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLabel" name="yminLabel">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>57</y>
        <width>32</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>y-min</string>
      </property>
      <property name="buddy">
       <cstring>yminDSpinBox</cstring>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="yminDSpinBox">
      <property name="geometry">
       <rect>
        <x>53</x>
        <y>57</y>
        <width>59</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>0.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="ymaxLabel">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>88</y>
        <width>36</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>y-max</string>
      </property>
      <property name="buddy">
       <cstring>ymaxDSpinBox</cstring>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="ymaxDSpinBox">
      <property name="geometry">
       <rect>
        <x>53</x>
        <y>88</y>
        <width>55</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="standardPushButton">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>119</y>
        <width>125</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Zoom To &amp;Standard</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="csGroupBox_2">
     <property name="title">
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="distRButton">
        <property name="text">
         <string>Distribution</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="cumRButton">
        <property name="text">
         <string>Cumulative
Distribution</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BinomialDistPlotter</class>
   <extends>QFrame</extends>
   <header>BinomialDistPlotter.hpp</header>
   <container>1</container>
   <slots>
    <signal>newXMin(double)</signal>
    <signal>newXMax(double)</signal>
    <signal>newYMin(double)</signal>
    <signal>newYMax(double)</signal>
    <signal>newXTicks(double)</signal>
    <signal>newYTicks(double)</signal>
    <slot>setXMin(double)</slot>
    <slot>setXMax(double)</slot>
    <slot>setYMin(double)</slot>
    <slot>setYMax(double)</slot>
    <slot>setXTicks(double)</slot>
    <slot>setYTicks(double)</slot>
    <slot>autoXTicks(bool)</slot>
    <slot>autoYTicks(bool)</slot>
    <slot>setStandardWindow()</slot>
    <slot>f1ToFile()</slot>
    <slot>toggleGrid(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>nHSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>nSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>nHSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nHSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>xmaxSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
